test_node_smoke.py

```
@@ -0,0 +1,287 @@
+        edge_sensitivity=0.5,
+        auto_adjust=False,
+        foreground_bias=0.7,
+        color_clusters=4,
+        edge_refinement=True,
+        dither_handling=False,
+        binary_threshold=128,
+        progress_reporting=True,
+    )
+    assert image_out.shape[0] == 2
+    assert mask_out.shape[0] == 2
+    assert isinstance(report, str) and "Batch" in report or "Total" in report
```

Ruff (0.15.11)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)